package.json

```diff
@@ -87,6 +86,7 @@
 		"phpegjs": "1.0.0-beta7",
 		"postcss-loader": "2.0.6",
 		"prismjs": "1.6.0",
+		"puppeteer": "1.1.1",
```

Comment:
There's been a 1.2.0 release since the PR was first opened. We should probably plan to upgrade.